schemas/eq_submission_schema.json

```diff
@@ -239,59 +238,120 @@
     },
     "data": {
       "$id": "#/properties/data",
-      "type":  "object",
+      "type": "object",
       "title": "Submitted Data",
-      "items": {
-        "$id": "#/properties/data/items",
-        "type": "object",
-        "title": "The answers submitted",
-        "required": [
-          "answer_id",
-          "value"
-        ],
-        "properties": {
-          "answer_id": {
-            "$id": "#/properties/data/items/properties/answer_id",
-            "type": "string",
-            "title": "Answer ID",
-            "default": "",
-            "examples": [
-              "accommodation-type-answer",
-              "first-name",
-              "ethnic-group-answer"
+      "required": [
+        "answers",
+        "lists"
+      ],
+      "properties": {
+        "answers": {
+          "$id": "#/properties/data/properties/answers",
+          "type": "array",
+          "title": "Answer",
+          "items": {
+            "$id": "#/properties/data/properties/answers/items",
+            "type": "object",
+            "title": "The answers submitted",
+            "required": [
+              "answer_id",
+              "value"
             ],
-            "pattern": "^(.*)$"
-          },
-          "value": {
-            "$id": "#/properties/data/items/properties/value",
-            "type": ["string", "array"],
-            "title": "The response value",
-            "description": "Can be a single string or multiple strings (e.g. for checkboxes)",
-            "pattern": "^(.*)$",
-            "items": {
-              "$id": "#/properties/data/items/properties/value/items",
-              "type": "string",
-              "title": "answer value items",
-              "default": "",
-              "examples": [
-                "Christian",
-                "Buddhist",
-                "Yes",
-                "No"
-              ],
-              "pattern": "^(.*)$"
+            "properties": {
+              "answer_id": {
+                "$id": "#/properties/data/properties/answers/items/properties/answer_id",
+                "type": "string",
+                "title": "Answer ID",
+                "default": "",
+                "examples": [
+                  "accommodation-type-answer",
+                  "first-name",
+                  "ethnic-group-answer"
+                ],
+                "pattern": "^(.*)$"
+              },
+              "value": {
+                "$id": "#/properties/data/properties/answers/items/properties/value",
+                "type": ["string", "array"],
+                "title": "The response value",
+                "description": "Can be a single string or multiple strings (e.g. for checkboxes)",
```

Comment:
Will this still validate multiple items? e.g. for an array, now that items has been removed I don't think this will validate?

Reply:
Added items to "value" section, thanks for help!